sequencing/ATACSeq/jobSubmission/8_collateStage3QCMatrics.sh

```diff
@@ -127,7 +127,7 @@ Output directory will be: ${PEAKCOUNTS}
 
 EOF
  
-	Rscript -e "rmarkdown::render(paste0(commandArgs(trailingOnly=TRUE)[1], '/collateCellTypeCheck.Rmd'), output_file=paste0(commandArgs(trailingOnly=TRUE)[2], '/stage3SummaryStats_all.html'))" "${RSCRIPTS_DIR}" "${PEAKCOUNTS}" "${CONFIGR}" "${SETPEAKS}"
+	Rscript -e "rmarkdown::render(paste0(commandArgs(trailingOnly=TRUE)[1], '/collateCellTypeCheck.Rmd'), output_file=paste0(commandArgs(trailingOnly=TRUE)[2], '/stage3SummaryStats_',commandArgs(trailingOnly=TRUE)[4],'.html'))" "${RSCRIPTS_DIR}" "${PEAKCOUNTS}" "${CONFIGR}" "${SETPEAKS}"
```

Comment:
Why are you passing arguments into an `Rscript` call that is being made on the command line? You could just replace this with:
```bash
Rscript -e "rmarkdown::render('${RSCRIPTS_DIR}/collateCellTypeCheck.Rmd'), output_file='${PEAKCOUNTS}/stage3SummaryStats_',commandArgs(trailingOnly=TRUE)[4],'.html'))" "${RSCRIPTS_DIR}" "${PEAKCOUNTS}" "${CONFIGR}" "${SETPEAKS}"
```

(not sure if you need "${RSCRIPTS_DIR}" and "${PEAKCOUNTS}" in the actual `.Rmd` file so I left these in)